src/accessibility/AccessibilityManager.js

```diff
@@ -305,9 +305,16 @@ AccessibilityManager.prototype.addChild = function(displayObject)
 	}
 
 
+	if(displayObject.accessibleTitle) {
+		div.title = displayObject.accessibleTitle;
+	} else if (!displayObject.accessibleTitle && !displayObject.accessibleHint) {
+		div.title = 'displayObject ' + this.tabIndex;
+	}
```

Comment:
This brace style is inconsistent with the rest of the lib, can we clean it up please?